lib/shared/store.js

```diff
@@ -28,6 +28,52 @@ const fileExtensions = require('./file-extensions')
 const utils = require('./utils')
 const packageJSON = require('../../package.json')
 
+/**
+ * @summary Verify and throw if any state fields are nil
+ * @function
+ * @public
+ *
+ * @param {Object} object - state object
+ * @param {Array<Array<String>> | Array<String>} fields - array of object field paths
```

Comment:
Is this line correct? Should it just be
```
 * @param {Array<String>} fields - array of object field paths
```
?

Reply:
It's correct, it just passes its argument to `_.get` which can take a string or an array of string for more complex path traversal.

Reply:
Ahh, then perhaps that behaviour ought to be unit-tested too then?